ingest/src/main/java/com/connexta/ingest/service/impl/IngestServiceImpl.java

```diff
@@ -29,16 +30,20 @@
 
   @NotNull private final S3StorageAdaptor s3Adaptor;
   @NotNull private final TransformClient transformClient;
-  private final String callbackEndpoint;
+  @NotEmpty private final String callbackEndpoint;
+  @NotEmpty private final String retrieveEndpoint;
 
   public IngestServiceImpl(
       @NotNull final S3StorageAdaptor s3Adaptor,
       @NotNull final TransformClient transformClient,
-      @Value("${endpointUrl.ingest.callback}") String callbackEndpoint) {
+      @NotEmpty @Value("${endpointUrl.ingest.callback}") final String callbackEndpoint,
```

Comment:
It doesn't have to be in this PR, but we should move the property injection out to the config file. Similar to how we do it in the S3Adaptor. This keeps our Spring stuff separated from our business logic.

Reply:
:+1: Let's address that in a different PR.